### ISSUE TITLE
test_id error

### ISSUE THREAD

**Issue by nOkuda** [tesserae/tesserae-v5#12]
Here the error message:

```
__________________________________________________________________________________________ test_id ___________________________________________________________________________________________

default_attrs = {'_id': None, 'form': None, 'frequency': None, 'text': None}
valid_attrs = [{'form': 'arma', 'frequency': 148, 'id': ObjectId('5bd0b96942facf27900cd402'), 'text': 'la/poetry/vergil.aeneid.tess'...': 'primus', 'frequency': 34, 'id': ObjectId('5bd0b96942facf27900cd407'), 'text': 'la/poetry/vergil.aeneid.tess'}, ...]

    def test_id(default_attrs, valid_attrs):
>       entity_id(Frequency, default_attrs, valid_attrs)

tests/db/entities/test_frequency.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

entity = <class 'tesserae.db.entities.frequency.Frequency'>, default_attrs = {'_id': None, 'form': None, 'frequency': None, 'text': None}
valid_attrs = [{'form': 'arma', 'frequency': 148, 'id': ObjectId('5bd0b96942facf27900cd402'), 'text': 'la/poetry/vergil.aeneid.tess'...': 'primus', 'frequency': 34, 'id': ObjectId('5bd0b96942facf27900cd407'), 'text': 'la/poetry/vergil.aeneid.tess'}, ...]

    def entity_id(entity, default_attrs, valid_attrs):
        e = entity()
        assert e.id == e._id
    
        for v in valid_attrs:
>           e = entity(id=v['_id'])
E           KeyError: '_id'

tests/db/entities/entity_test_functions.py:59: KeyError
```

I'm guessing once we fix this up, we'll also fix the error with test_json_encode, which also comes up with a KeyError on `_id`.

**Comment by jeffkinnison** [comment]
I believe so, yes. There's some other location in the unit tests that isn't properly restoring '_id' into the test data, I'm looking for it now.

**Comment by nOkuda** [comment]
Probably fixed back in #8.
